packer/linux/scripts/install-docker.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-DOCKER_COMPOSE_V2_VERSION=2.24.4
-DOCKER_BUILDX_VERSION=0.12.1
+DOCKER_COMPOSE_V2_VERSION=2.24.6
```

Comment:
While I'm here, also bump Docker Compose a couple of patch versions to 2.24.6.

Release notes:
- [2.24.6](https://github.com/docker/compose/releases/tag/v2.24.6)
- [2.24.5](https://github.com/docker/compose/releases/tag/v2.24.5)